modules/authentication/src/models/User.schema.ts

```diff
@@ -39,6 +39,14 @@ const schema: ConduitModel = {
     type: TYPE.String,
     required: false,
   },
+  ethPublicAddress: {
```

Comment:
add the metamask fields under a dedicated metamask field, like we do with the oAuth providers